src/mavsdk_server/src/plugins/camera/camera_service_impl.h

```diff
@@ -713,6 +804,17 @@ class CameraServiceImpl final : public rpc::camera::CameraService::Service {
 
         obj.vertical_resolution_px = information.vertical_resolution_px();
 
+        obj.lens_id = information.lens_id();
+
+        obj.definition_file_version = information.definition_file_version();
+
+        obj.definition_file_uri = information.definition_file_uri();
+
+        for (const auto& elem : information.camera_cap_flags()) {
+            obj.camera_cap_flags.push_back(translateFromRpcCameraCapFlags(
+                static_cast<mavsdk::rpc::camera::Information::CameraCapFlags>(elem)));
```

Comment:
the auto generate code is static_cast<mavsdk::rpc::camera::CameraCapFlags> missing Information domain. I just manual add **Information** in auto generate code.